assets/000-postgres.prep

```diff
@@ -52,6 +53,9 @@ ifthenelse { s6-test -d "${PGDATA}/base" -a -f "${PGDATA}/PG_VERSION" }
   }
 }
 }
+foreground {
+  sed -i "s/max_connections = 100/max_connections = 1024/g" ${PG_CONF_PATH}
```

Comment:
Any chance we can do this at image build time?

Reply:
I tried https://github.com/pulp/pulp_ansible/pull/599
but when you try to restart postgres it crashes

Reply:
OH, i meant somewhere in the Containerfile but maybe that file is only created on the fly.